src/components/application_manager/rpc_plugins/sdl_rpc_plugin/include/sdl_rpc_plugin/commands/mobile/perform_interaction_request.h

```diff
@@ -241,6 +289,8 @@ class PerformInteractionRequest
   hmi_apis::Common_Result::eType ui_result_code_;
   std::string ui_info_;
   std::string vr_info_;
+  smart_objects::SmartObject vr_params_;
+  FirstAnsweredInterface first_responser_;
```

Comment:
`responser` -> `responder` everywhere :)

Reply:
@iCollin Renamed in f35543c901a896b89a2206e4bd57b6b213a7c5aa